tableauserverclient/models/user_item.py

```diff
@@ -184,6 +186,18 @@ def groups(self) -> "Pager":
             raise UnpopulatedPropertyError(error)
         return self._groups()
 
+    @property
+    def idp_configuration_id(self) -> Optional[str]:
+        """
+        IDP configuration id for the user. This is only available on Tableau
+        Cloud, 3.24 or later
+        """
+        return self._idp_configuration_id
+
+    @idp_configuration_id.setter
+    def idp_configuration_id(self, value: str) -> None:
+        self._idp_configuration_id = value
+
```

Comment:
While this is currently mutually exclusive with `TSC.UserItem.auth_setting`, @sf-stikone and I discussed leaving that out of TSC, instead leaving validation to occur server-side in case that should change in the future.